src/Servers/Kestrel/Transport.Sockets/src/Internal/TransferResult.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Sockets;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal
+{
+    internal readonly struct TransferResult
```

Comment:
Maybe this should just be a ValueTuple

Reply:
As a base class or as-is? I wouldn't like to see the `ValueTuple<>` name everywhere it's used, not readable.

Reply:
Does "Maybe" mean "please change" ?